baseapp/baseapp.go

```diff
@@ -131,6 +133,12 @@ func (app *BaseApp) SetCommitMultiStoreTracer(w io.Writer) {
 	app.cms.SetTracer(w)
 }
 
+// AppVersion returns the application's version string.
+func (app *BaseApp) AppVersion() string { return app.appVersion }
+
+// SetAppVersion sets the application's version string.
+func (app *BaseApp) SetAppVersion(v string) { app.appVersion = v }
```

Comment:
This should probably live in `options.go` and check for sealed?

Reply:
++

Reply:
Done, thanks